.gitignore

```diff
@@ -7,6 +7,9 @@ yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
 
+# Package lock
```

Comment:
Why would you want to exclude the `package-lock.json`? This file is important for having the same versions across installs:

https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5